DEVELOPMENT-GUIDE.md

```diff
@@ -0,0 +1,85 @@
+# ACCP Development Guidance
+Like all cryptographic implementations, correctness and code-safety is paramount in the Amazon Corretto Crypto Provider.
```

Comment:
I would recommend making this more direct, remove "Like all cryptographic implementations".